*** 2016-05-23 13:57:02.98 idempiere Log (CLogFile) ***
13:57:02.980 Adempiere.startup: iDempiere(r) Release 3.1_2015-10-31 -Smart Suite ERP, CRM and SCM- (c) 1999-2013 iDempiere(r); Implementation: iDempiere - Supported by iDempiere community [14]
13:57:02.980 Ini.loadProperties: E:\mercurialidempiere/idempiere-test.properties #32 [14]
13:57:03.320 DB.isBuildOK: Build DB=${env.ADEMPIERE_VERSION} 20080428-1232 [14]
13:57:03.320 DB.isBuildOK: Build Cl=3.1.0.qualifier [14]
13:57:03.336-----------> DB.isBuildOK: Build Version Error 

The program assumes build version 3.1.0.qualifier, but database has build version ${env.ADEMPIERE_VERSION} 20080428-1232. 
This is likely to cause hard to fix errors. 
Please contact administrator. [14]
13:57:03.414 Language.getLanguage: Adding Language=en, Country=US, Locale=en_US [14]
13:57:03.429 Language.getLanguage: Adding Language=en, Country=US, Locale=en_US [14]
13:57:03.476 WebEnv.initWeb: Servlet Context Init Parameters: iDempiere Root
ServerRoot=1 [14]
13:57:03.492-----------> EMail.setFrom: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [14]
13:57:03.492-----------> EMail.addTo: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [14]
13:57:03.492-----------> EMail.isValid: From is invalid=null [14]
13:57:03.492-----------> EMail.isValid: From is invalid=null [14]
13:57:03.492-----------> MClient.sendEMail: Could NOT Send Email: Server started: iDempiere (odbz:10.0.5.43) to you @ company.org: Invalid Data (System) [14]
13:57:03.492 WebEnv.initWeb: Servlet Init Parameter: idempiereMonitor [14]
13:57:03.492 AdempiereMonitor.init:  [14]
13:57:04.954-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [14]
13:57:05.209-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://178.fwk333967374:2/metainfo/zk/config.xml
with bundleresource://178.fwk333967374:19/metainfo/zk/config.xml [14]
13:57:05.212-----------> ClassLocator.getDependentXMLResources: Replicate resource: zk
Overwrite bundleresource://178.fwk333967374:11/metainfo/zk/config.xml
with bundleresource://178.fwk333967374:28/metainfo/zk/config.xml [14]
13:57:05.216-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://178.fwk333967374:12/metainfo/zk/config.xml
with bundleresource://178.fwk333967374:29/metainfo/zk/config.xml [14]
13:57:05.237-----------> ClassLocator.getDependentXMLResources: Replicate resource: zul
Overwrite bundleresource://178.fwk333967374:14/metainfo/zk/zk.xml
with bundleresource://178.fwk333967374:31/metainfo/zk/zk.xml [14]
13:57:05.556-----------> LanguageDefinition.<init>: Replicated language: xhtml, overriden by [LanguageDefinition: xhtml] [14]
13:57:05.570-----------> LanguageDefinition.<init>: Replicated language: xul/html, overriden by [LanguageDefinition: xul/html] [14]
13:57:05.633-----------> ClassLocator.getDependentXMLResources: Replicate resource: calendar
Overwrite bundleresource://178.fwk333967374:1/metainfo/zk/lang-addon.xml
with bundleresource://178.fwk333967374:18/metainfo/zk/lang-addon.xml [14]
13:57:05.633-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://178.fwk333967374:2/metainfo/zk/lang-addon.xml
with bundleresource://178.fwk333967374:19/metainfo/zk/lang-addon.xml [14]
13:57:05.633-----------> ClassLocator.getDependentXMLResources: Replicate resource: gmapsz
Overwrite bundleresource://178.fwk333967374:5/metainfo/zk/lang-addon.xml
with bundleresource://178.fwk333967374:22/metainfo/zk/lang-addon.xml [14]
13:57:05.633-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeline
Overwrite bundleresource://178.fwk333967374:6/metainfo/zk/lang-addon.xml
with bundleresource://178.fwk333967374:23/metainfo/zk/lang-addon.xml [14]
13:57:05.633-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeplot
Overwrite bundleresource://178.fwk333967374:7/metainfo/zk/lang-addon.xml
with bundleresource://178.fwk333967374:24/metainfo/zk/lang-addon.xml [14]
13:57:05.648-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://178.fwk333967374:12/metainfo/zk/lang-addon.xml
with bundleresource://178.fwk333967374:29/metainfo/zk/lang-addon.xml [14]
13:57:05.664-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkplus
Overwrite bundleresource://178.fwk333967374:13/metainfo/zk/lang-addon.xml
with bundleresource://178.fwk333967374:30/metainfo/zk/lang-addon.xml [14]
13:57:05.753           WebUIServlet.init: iDempiere Web Client started successfully [14]
13:57:06.701-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [14]

*** 2016-05-23 13:57:06.934 idempiere Log (CLogFile) ***
13:57:08.544-----------> Node: [10.0.5.43]:5701 [adempiere.idempiere/localhost] [3.5.3] No join method is enabled! Starting standalone. [46]
13:58:06.359===========> MRole.addAccessSQL: TableName not correctly parsed - TableNameIn=C_Invoice - AccessSqlParser[C_Invoice=WHERE|0] - #1 C_Invoice=WHERE
 = SELECT C_Invoice_ID, DocumentNo, DateInvoiced, GrandTotal, Balance FROM C_Invoice WHERE  [26]
14:48:17.415-----------> BasicResourcePool.log: com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@799ee03e -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (2). Last acquisition attempt exception:  [43]
org.postgresql.util.PSQLException: El intento de conexión falló.; State=08001; ErrorCode=0
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:257)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:65)
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:149)
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:35)
	at org.postgresql.jdbc3g.AbstractJdbc3gConnection.<init>(AbstractJdbc3gConnection.java:22)
	at org.postgresql.jdbc4.AbstractJdbc4Connection.<init>(AbstractJdbc4Connection.java:47)
	at org.postgresql.jdbc4.Jdbc4Connection.<init>(Jdbc4Connection.java:30)
	at org.postgresql.Driver.makeConnection(Driver.java:414)
	at org.postgresql.Driver.connect(Driver.java:282)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:200)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1127)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1114)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$800(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1855)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
caused by: java.net.NoRouteToHostException: No route to host: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at org.postgresql.core.PGStream.<init>(PGStream.java:61)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:129)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:65)
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:149)
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:35)
	at org.postgresql.jdbc3g.AbstractJdbc3gConnection.<init>(AbstractJdbc3gConnection.java:22)
	at org.postgresql.jdbc4.AbstractJdbc4Connection.<init>(AbstractJdbc4Connection.java:47)
	at org.postgresql.jdbc4.Jdbc4Connection.<init>(Jdbc4Connection.java:30)
	at org.postgresql.Driver.makeConnection(Driver.java:414)
	at org.postgresql.Driver.connect(Driver.java:282)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)

14:48:17.465-----------> BasicResourcePool.log: Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@228779e2 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests. [43]
14:48:15.412===========> PreparedStatementProxy.init: SELECT COUNT(1) FROM AD_Note WHERE AD_Client_ID=? AND AD_User_ID IN (0,?) AND Processed='N' AND AD_BroadcastMessage_ID IS NULL [24]
java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getNoticeCount(DPActivitiesModel.java:55)

14:48:34.330===========> DB.getSQLValue: SELECT COUNT(1) FROM AD_Note WHERE AD_Client_ID=? AND AD_User_ID IN (0,?) AND Processed='N' AND AD_BroadcastMessage_ID IS NULL [24]
org.adempiere.exceptions.DBException: java.lang.IllegalStateException: DB.getConnectionRO - No hay conexión con la base de datos
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:73)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getNoticeCount(DPActivitiesModel.java:55)
	at org.adempiere.webui.dashboard.DPActivities.refresh(DPActivities.java:126)
	at org.adempiere.webui.dashboard.DashboardRunnable.refreshDashboard(DashboardRunnable.java:131)
	at org.adempiere.webui.dashboard.DashboardRunnable.run(DashboardRunnable.java:92)
	at org.adempiere.webui.desktop.DashboardController$3.onEvent(DashboardController.java:461)
	at org.zkoss.zk.ui.AbstractComponent.onEvent(AbstractComponent.java:2773)
	at org.zkoss.zk.ui.AbstractComponent.service(AbstractComponent.java:2744)
	at org.zkoss.zk.ui.AbstractComponent.service(AbstractComponent.java:2685)
	at org.zkoss.zk.ui.impl.EventProcessor.process(EventProcessor.java:136)
	at org.zkoss.zk.ui.impl.UiEngineImpl.processEvent(UiEngineImpl.java:1765)
	at org.zkoss.zk.ui.impl.UiEngineImpl.process(UiEngineImpl.java:1550)
	at org.zkoss.zk.ui.impl.UiEngineImpl.execUpdate(UiEngineImpl.java:1260)
	at org.zkoss.zk.au.http.DHtmlUpdateServlet.process(DHtmlUpdateServlet.java:611)
caused by: java.lang.IllegalStateException: DB.getConnectionRO - @NoDBConnection@
	at org.compiere.util.DB.createConnection(DB.java:470)
	at org.idempiere.db.util.AutoCommitConnectionBroker.getConnection(AutoCommitConnectionBroker.java:48)
	at org.compiere.db.PreparedStatementProxy.init(PreparedStatementProxy.java:64)
	at org.compiere.db.PreparedStatementProxy.<init>(PreparedStatementProxy.java:45)
	at org.compiere.db.ProxyFactory.newCPreparedStatement(ProxyFactory.java:56)
	at org.compiere.util.DB.prepareStatement(DB.java:773)
	at org.compiere.util.DB.prepareStatement(DB.java:742)
	at org.compiere.util.DB.getSQLValueEx(DB.java:1276)
	at org.compiere.util.DB.getSQLValue(DB.java:1321)
	at org.compiere.util.DB.getSQLValue(DB.java:2187)
	at org.adempiere.webui.dashboard.DPActivitiesModel.getNoticeCount(DPActivitiesModel.java:55)

